packages/redux/MIGRATION.md

```diff
@@ -590,24 +593,19 @@ The following tables contain the exports that were removed and the alternatives
 
 ##### Action types
 
-| Removed export                                | Notes                                |
-| --------------------------------------------- | ------------------------------------ |
-| actionTypes.POST_CHARGES\*¹                   | Removed as its action was deprecated |
-| actionTypes.COMPLETE_PAYMENT_CHECKOUT\*¹      | Removed as its action was deprecated |
-| actionTypes.COMPLETE_PAYMENT_CHECKOUT\*¹      | Removed as its action was deprecated |
-| actionTypes.UPDATE_GIFT_MESSAGE\*¹            | Removed as its action was deprecated |
-| actionTypes.UPDATE_DELIVERY_BUNDLE_UPGRADE\*¹ | Removed as its action was deprecated |
+| Removed export                     | Notes                                |
+| ---------------------------------- | ------------------------------------ |
+| actionTypes.POST_CHARGES\*¹        | Removed as its action was deprecated |
```

Comment:
Since the doPostCharges action was renamed, then these action types were as well, right?

Reply:
I made a mistake and swap these two actions: doPostCharges and doCompletePaymentCheckout
Fixed now:
actionTypes.POST_CHARGES -> checkoutActionTypes.CREATE_CHECKOUT_ORDER_CHARGE

doCompletePaymentCheckout and actionTypes.COMPLETE_PAYMENT_CHECKOUT was deprecated